mesheryctl/pkg/utils/fixtures/listmanifest.api.response.golden

```diff
@@ -0,0 +1,182 @@
+{
+  "sha": "47c634a49e6d143a54d734437a26ad233146ddf5",
+  "url": "https://api.github.com/repos/meshery/meshery/git/trees/47c634a49e6d143a54d734437a26ad233146ddf5",
+  "tree": [
+    {
+      "path": "k8s-1-16-deployments",
+      "mode": "040000",
```

Comment:
can you compress these files in one-liner json? if possible

Reply:
any reason for it? I mean file size will stay the same.

Reply:
Just that it will look smaller!

Reply:
an extra function will have to be added to minify this. Or I will have to statically minify these and add there.
I think we should leave this as it is. Plus this has some kind readablity in case of some issue. 
Otherwise, whatever you say